README.md

```diff
@@ -5,12 +5,30 @@ parameters in a global, hierarchical key database._
 
 <img src="https://cdn.rawgit.com/ElektraInitiative/libelektra/master/doc/images/circle.svg" alt="Elektra" width="50" />
 
-The core is a small library implemented in C. The plugin-based framework fulfills many
-configuration-related tasks to avoid any unnecessary code duplication
-across applications while it still allows the core to stay without any
-external dependency. Elektra abstracts from cross-platform-related issues
-with a consistent API, and allows applications to be aware of other
-applications' configurations, leveraging easy application integration.
+Elektra provides a simple API for applications to access and update configuration 
```

Comment:
I guess I'm a bit biased against the word generic. How about "mature and easily comprehensible", but if you still prefer generic, I'll adopt it.